### ISSUE TITLE
Thread Safe and Update

### ISSUE THREAD

**Issue by melloware** [cleydyr/dart-sass-maven-plugin#21]
@cleydyr just a couple of things.

1. I see this warning in my Parrallel execution of tests but I am not having any issues just thought you should know.

```
Warning:  *****************************************************************
Warning:  * Your build is requesting parallel execution, but this         *
Warning:  * project contains the following plugin(s) that have goals not  *
Warning:  * marked as thread-safe to support parallel execution.          *
Warning:  * While this /may/ work fine, please look for plugin updates    *
Warning:  * and/or request plugins be made thread-safe.                   *
Warning:  * If reporting an issue, report it against the plugin in        *
Warning:  * question, not against Apache Maven.                           *
Warning:  *****************************************************************
Warning:  The following plugins are not marked as thread-safe in PrimeFaces Showcase:
Warning:    io.github.cleydyr:dart-sass-maven-plugin:0.3.0
Warning:  
Warning:  Enable debug to see precisely which goals are not marked as thread-safe.
Warning:  *****************************************************************
```

2. I wonder if we should update to the latest Dart SASS Exe's?

3. Finally I have been using this for months without issue maybe time for a 1.0.0 release?

**Comment by cleydyr** [comment]
Hey, @melloware .

Thanks for the feedback! I'm glad to know that this plugin has been useful. And yeah: it's already time for a 1.0.0 release. 🙌 

I'll update the plugin with the latest Dart Sass executables soon and make a new release.

Regarding the thread-safe part, I'll have look through the code to figure out if I can mark the plugin goals as thread-safe.

**Comment by melloware** [comment]
You rock!!

**Comment by cleydyr** [comment]
Hey, @melloware. The only part of the plugin execution unsafe for multi-threading is the phase where the plugin potentially extracts files to a temporary directory so that it can run the executable afterward. If the files are not there and two threads try to write data to the same file, we may run into problems. Other than that, the plugin can be safely used for parallel builds. So I'll `synchronize` the potentially unsafe part and give it a go at marking the plugin thread-safe.

Thoughts?

**Comment by melloware** [comment]
That sounds like a good idea to me.

**Comment by cleydyr** [comment]
It will be included in version 1.1.0. Closing this.
